actionpack/test/controller/mime/respond_to_test.rb

```diff
@@ -157,6 +157,13 @@ def handle_any
     end
   end
 
+  def handle_any_doesnt_set_request_content_type
+    respond_to do |type|
+      type.html { render body: "HTML" }
+      type.any { render json: { foo: "bar" } }
```

Comment:
what happens if you try to render a template?

Reply:
If we render a template, there is 2 possibilities but the response content-type will be always based on the request format.

1) In case a template handles a particular format
```csv.erb
# filename: foo.csv.erb

foo,bar,baz
1,2,3
```

```ruby
type.any { render 'foo' } 
# Response's content type will be `text/csv`
```

2) In case a template doesn't handle a particular format
```
# filename: foo.erb

Hello world
```

```ruby
type.any { render 'foo' } 
# Response's content type will be `text/csv`
```